rich/progress.py

```diff
@@ -299,7 +299,9 @@ def render(self, task: "Task") -> Text:
         completed = int(task.completed)
         total = int(task.total)
         unit, suffix = filesize.pick_unit_and_suffix(
-            total, ["bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"], 1024
+            total,
```

Comment:
I think filesize should be in decimal units, to be consistent with file managers. i.e. same units, just with the third parameter as 1000.

Reply:
Sure let me change that